Dynatrace/dynatrace-operator
# Add monitored entity reconciler

## Description

With this a new reconciler is added that deals with the monitored entity from the cluster. It will be stored in the dynakube status from now on, such that if we need to store the value we can guarantee that it is stored somewhere.

Because of this I removed the API call which was inside the settings enrichment reconciler, as it is unneeded as the main purpose was to get the entityID. From now on the ID is simply taken from the dynakube status.

JIRA-ticket: https://dt-rnd.atlassian.net/browse/K8S-9857



## How can this be tested?

Deploy the operator and a dynakube with metadata enrichment enabled. Check the logs, also check the dynakube status.
